ts/components/ChooserListComponent.tsx

```diff
@@ -0,0 +1,105 @@
+import I18n from "i18n-js";
+import { Container, Content, View } from "native-base";
+import * as React from "react";
+import { FlatList, ListRenderItem, StyleSheet } from "react-native";
+import customVariables from "../theme/variables";
+import AppHeader from "./ui/AppHeader";
+import FooterWithButtons from "./ui/FooterWithButtons";
+
+type Props = {
+  items: ReadonlyArray<any>;
```

Comment:
Do not use "any". Try to create a [Generic type](https://www.typescriptlang.org/docs/handbook/generics.html).

Reply:
@fpersico Done in commit 835dd12